bin/go-rpm-integration

```diff
@@ -213,17 +213,24 @@ for dir in $(\
 done
 }
 
+fullprovides(){
+  namespace=${1}
+  prov=${2}
+  for index in "${!deco[@]}" ; do
+    echo "${namespace}(${prov})${deco[index]}${version:+ = ${version}}"
+  done
+}
+
 provides() {
 goipath="${1}"
+fullprovides golang-ipp "${goipath}"
```

Comment:
+1, this approach is better. We can provide Go packages in more namespaces depending on their semantics.

Reply:
do you also want golang-goipath as namespace or do you prefer golang-ipp?

Reply:
Yeah, it crossed my mind. `golang-ipp` does not look shiny, `golang-goipath` has the `go` twice in its name, thought `golang-ipath` does not look like a superstar either. We discussed with @jcajka this path as well. He plans to bridge the `go get` at some point so it pulls rpm instead of repositories (in case the configuration allows so). So `go get` would search for all paths that are provided in both `golang` and `golang-???` namespace to check if the import path prefix exists at all. So the `golang-???` name should reflect it is gettable as well. `golang-you-can-get-me` is too much. So at this point I am fine with both `golang-ipp` and `golang-goipath`. Maybe the latter looks better than the first one. The first one looks like a rabbit with no ears.